src/modules/usenet/index.ts

```diff
@@ -0,0 +1,2 @@
+export { UsenetModule } from './UsenetModule';
+export * from './types';
```

Comment:
Are you sure this doesn't break the module enabler ? 😅
It's not needed as you already export it in the file below but i understand why you would've done it like this

Reply:
Tbh I'm not a fan of barrels but I figured since you're using them I might as well 😄